solutions/deploy-argocd-apps-pull/getting-started.md

```diff
@@ -0,0 +1,123 @@
+# Getting Started
+
+### Prerequisites
+- [kind](https://kind.sigs.k8s.io) must be installed on your local machine. The Kubernetes version must be >= 1.19, see [kind user guide](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster) for more details.
+- 
+
+### Steps
+1. Setup an OCM Hub cluster and registered an OCM Managed cluster. 
+   ```
+   curl -L https://raw.githubusercontent.com/open-cluster-management-io/OCM/main/solutions/setup-dev-environment/local-up.sh | bash
+   ```
+   
+   See [Open Cluster Management Quick Start](https://open-cluster-management.io/getting-started/quick-start/) for more details.
+
+2. Install ArgoCD on the hub cluster and both managed clusters. 
+   ```
+   kubectl config use-context kind-hub #repeat for kind-cluster1 and kind-cluster2
+   kubectl create namespace argocd
+   kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+   ```
+   See [ArgoCD website](https://argo-cd.readthedocs.io/en/stable/getting_started/) for more details.
+
+3. On the Hub cluster, scale down the Application controller:
```

Comment:
We don't need to mention this anymore because the feature is now in the latest stable version of ArgoCD.